### DISCUSSION TITLE
Hayward Aquarite not responding ok to status

### DISCUSSION THREAD

**Discussion by BigDaddyFL** [tagyoureit/nodejs-poolController#955] [Q&A] [answered]
I  have an AquaRite SWG.  I have connected it via an Elfin EW11A and responding to the app.  The board I have in the SWG is a replacement board from when I had a catastrophic failure and came back to a green pool.   
I have the setup complete and the SWG is responding but with a not OK message.  The remotely controlled light is on but the status message is not ok.  I am struggling here.  I have removed and re-setup the SWG in the dashboard but stilll no luck.  Here is the output from the message manager.

![image](https://github.com/tagyoureit/nodejs-poolController/assets/171095320/43498edc-e37e-45a6-8491-b5d7dfa01b76)

 and here is the decode of the status response:

![image](https://github.com/tagyoureit/nodejs-poolController/assets/171095320/631dac43-0af5-4296-b197-b97162778c7f)

What can I check to find out why the response is 0,0?

**Comment by BigDaddyFL** [comment]
I also confirmed that when it is not remotely controlled the SWG is working correctly.

**Reply by BigDaddyFL** [reply]
According to [Cholrinator Info](https://github.com/tagyoureit/nodejs-poolController/wiki/Chlorinator#this-page-is-for-the-pentair-and-hayward-aka-aquarite-naturesoft-mineralsprings-smartpure-etc-salt-chlorinator) the response is as follows:

![image](https://github.com/tagyoureit/nodejs-poolController/assets/171095320/62e939a7-4548-46db-b2ab-bd2aa60620be)

Shouldn't byte 4 be taken as the OK in the response instead of 5 and 6 for Hayward?  This is what shows in the dash panel:

![image](https://github.com/tagyoureit/nodejs-poolController/assets/171095320/a96ccd97-d8af-4cc6-a08a-3fc621a40348)

should the light be green since it is communicating?

**Reply by tagyoureit** [reply]
The light indicates if it is on (above 0%), not if it is communicating.

**Reply by BigDaddyFL** [reply]
I have it set to 55% but i do not see that sent in the message manager.  I have also tried to change the value in the dashboard to a different percent and also set the super chlorinate on but nothing is reflected in the messages to represent this change.  Is there something I need to enable or does the nixie not allow this.

I can find alot of info on jandy but not on Hayward SWGs

I apologize if I am misunderstanding but the docs are less than clear for me.

**Reply by tagyoureit** [reply, marked as answer]
Your main Filter (pool or spa) circuit needs to be on  and the chlorinator needs to be set up in that same body.

**Comment by BigDaddyFL** [comment]
That fixed it.  Thanks for the assistance.
